bot/constants.py

```diff
@@ -5,9 +5,14 @@
 PYTHON_GUILD = 267624335836053506
 
 BOT_CHANNEL = 267659945086812160
+HELP1_CHANNEL = 303906576991780866
+HELP2_CHANNEL = 303906556754395136
+HELP3_CHANNEL = 303906514266226689
+PYTHON_CHANNEL = 267624335836053506
 DEVLOG_CHANNEL = 409308876241108992
 VERIFICATION_CHANNEL = 352442727016693763
 
+
```

Comment:
why is this here

Reply:
This line is unacceptable. Requested changes.